scripts/createImage.sh

```diff
@@ -17,9 +18,47 @@ dependencies=(
     target/dependency/armada-scala-client_${SCALA_BIN_VERSION}-0.1.0-SNAPSHOT.jar
 )
 
+spark_v3_dependencies=(
+    target/dependency/grpc-api-1.47.1.jar
+    target/dependency/grpc-core-1.47.1.jar
+    target/dependency/grpc-netty-1.47.1.jar
+    target/dependency/grpc-protobuf-1.47.1.jar
+    target/dependency/grpc-context-1.47.1.jar
+    target/dependency/grpc-stub-1.47.1.jar
+    target/dependency/guava-31.0.1-android.jar
+    target/dependency/failureaccess-1.0.1.jar
+    target/dependency/perfmark-api-0.25.0.jar
+)
+
+spark_v334_dependencies=(
+    target/dependency/netty-codec-http2-4.1.72.Final.jar
+    target/dependency/netty-codec-http-4.1.72.Final.jar
+    target/dependency/protobuf-java-3.19.6.jar
+)
+
+if [[ $SPARK_VERSION == 3* ]]; then
+    dependencies+=("${spark_v3_dependencies[@]}")
+fi
+
+if [[ $SPARK_VERSION == "3.3.4" ]]; then
+    dependencies+=("${spark_v334_dependencies[@]}")
+fi
+
+
+# Replace these with newer versions below
```

Comment:
Why is this needed? Does Spark pull in different versions than needed for `armada-scala-client`?

Reply:
yes, those versions don't work with the armada client.  I'll add a comment to clarify.